.github/workflows/release.yml

```diff
@@ -389,6 +390,26 @@ jobs:
             workspace/runtime-*.yml
             workspace/emojivoto-demo.yml
             workspace/mysql-demo.yml
+      - name: AWS login (IAM role)
+        uses: aws-actions/configure-aws-credentials@f24d7193d98baebaeacc7e2227925dd47cc267f5 # v4.2.0
+        with:
+          role-to-assume: arn:aws:iam::795746500882:role/ContrastPublicBucketRW
+          aws-region: eu-central-1
+      - name: Upload pre-release artifacts to S3 bucket contrast-public
+        run: |
+          unix=$(date +%s)
+          files=(
+            workspace/contrast-cli/bin/contrast
+            workspace/contrast-enterprise-cli/bin/contrast-enterprise
+            workspace/coordinator*.yml
+            workspace/runtime-*.yml
+            workspace/emojivoto-demo.yml
+            workspace/mysql-demo.yml
+          )
+          for file in "${files[@]}"; do
+            aws s3 cp "$file" "s3://contrast-public/pre-releases/${{ inputs.version }}/$unix/"
+          done
+          echo "Artifacts available at https://contrast-public.s3.eu-central-1.amazonaws.com/releases/${{ inputs.version }}/$unix/" | tee -a "$GITHUB_STEP_SUMMARY"
```

Comment:
Is this expected to be publicly accessible? I tried the URL from the test run, but got a permission denied for the directory as well as for individual files.

Reply:
Sorry, I tested the delete command, so now the files are gone. But I verified that they can be curled.